docs/event-serializing.md

```diff
@@ -0,0 +1,139 @@
+---
+id: event-serializing
+title: Event Serializing
+sidebar_label: Event Serializing
+---
+
+We support [Apache Avro](https://avro.apache.org/) and standard JSON (de)serialization of events for producer as well as for consumer. This is working together with [Cloud Events specification](https://github.com/cloudevents/spec) (versions `0.1` and `0.2`). To make both of these worlds work we are utilizing Kafka headers introduced in version `0.11`. As Cloud Events specify there are 2 modes how to make this combination work. Discussion about this topic can be followed on [Github](https://github.com/cloudevents/spec/pull/337/files).
+
+**Structured content mode**
+
+This mode should be used when JSON format is desired, thus no further serialization and deserialization such as Apache Avro.
+
+> ##### KAFKA EVENT HEADERS
+>
+> ```
+> cloudEvent_contentType: "application/json"
+> ```
+>
+> ##### KAFKA EVENT VALUE
+>
+> ```
+> {
+>    "specversion" : "0.1",
+>    "type" : "com.example.someevent",
+>    "data": {},
+>    ...
+> }
+> ```
+
+**Binary content mode**
+
+This mode should be used when further serialization and deserialization is desired (e.g. Apache Avro).
+
+> ##### KAFKA EVENT HEADERS
+>
+> ```
+> cloudEvent_contentType: "avro/binary"
+> cloudEvent_specversion: "0.2"
+> cloudEvent_type: "com.example.someevent"
```

Comment:
again, we should only use the newer versions here (e.g., ce-contenttype, ce-specversion, ce-type). If we support both that's fine, but in the examples I feel we should be current.

Reply:
Can you provide source where they changed it? Checked PR that's referenced in docs and github, but no clear visibility, or I'm blind.